src/tests/ftest/osa/osa_offline_drain.py

```diff
@@ -22,12 +25,30 @@ def setUp(self):
         """Set up for test case."""
         super().setUp()
         self.dmg_command = self.get_dmg_command()
+        self.daos_command = DaosCommand(self.bin)
+        self.ranks = self.params.get("rank_list", '/run/test_ranks/*')
+        self.test_oclass = self.params.get("oclass", '/run/test_obj_class/*')
         self.ior_test_sequence = self.params.get(
             "ior_test_sequence", '/run/ior/iorflags/*')
         # Recreate the client hostfile without slots defined
         self.hostfile_clients = write_host_file(
             self.hostlist_clients, self.workdir, None)
 
+    def simple_drain_reintegrate_loop(self, rank, loop_time=100):
+        """This method performs drain and reintegration on a rank,
```

Comment:
You might want to add argument description.

Reply:
Will do.